content/en/Product Updates/release-notes-august-2024.md

```diff
@@ -37,3 +36,35 @@ The problem we identified was that targets were used in different parts of the a
 To address this issue, we have introduced a new field called "Associated Targets" for web or API-related assets. This feature allows customers to directly associate targets with their assets and includes the option to add targets not listed in the Attack Surface using free text. Any targets added to an asset will also be included in new pentests created for that asset. Furthermore, pentesters can now select which targets in scope for a pentest are related to specific findings, allowing for more detailed attribution at the target level.
 
 ![Create Asset - Add Associated Targets](/release-notes/associated-targets.png "Add Associated Targets")
+
+---
+
+#### Enhanced SAML Configuration Page in Org Settings
+
+Customers utilizing Service Provider initiated SAML SSO can now seamlessly access this feature directly within the platform. After configuring SAML settings, organizations will now display a Cobalt Sign In URL. 
+
+This URL simplifies the user experience by automatically redirecting users to their designated SAML identity provider. For convenient access, users relying on SAML SSO for sign-in can now bookmark this link and effortlessly sign in to Cobalt. 
+
+![Create Asset - Add Associated Targets](/release-notes/saml-updates.png "SAML Updates")
```

Comment:
Just wanted to make sure the IdP details (URL with trexcloud.com) used in the image are fake.

I would also update the name of each image in brackets to match the name in quotation marks e.g. [SAML Updates]..."SAML Updates"

Reply:
Updated the image to blur the IDp SSO string.